Lib/Azure Storage Client Library/Azure Storage Client Library.xcodeproj/project.pbxproj

```diff
@@ -21,7 +21,27 @@
 /* End PBXAggregateTarget section */
 
 /* Begin PBXBuildFile section */
+		693220321C16437100B07D98 /* AZSSharedAccessHeaders.h in Headers */ = {isa = PBXBuildFile; fileRef = 6932202C1C16437100B07D98 /* AZSSharedAccessHeaders.h */; };
+		693220331C16437100B07D98 /* AZSSharedAccessHeaders.m in Sources */ = {isa = PBXBuildFile; fileRef = 6932202D1C16437100B07D98 /* AZSSharedAccessHeaders.m */; };
+		693220341C16437100B07D98 /* AZSSharedAccessPolicy.h in Headers */ = {isa = PBXBuildFile; fileRef = 6932202E1C16437100B07D98 /* AZSSharedAccessPolicy.h */; };
+		693220351C16437100B07D98 /* AZSSharedAccessPolicy.m in Sources */ = {isa = PBXBuildFile; fileRef = 6932202F1C16437100B07D98 /* AZSSharedAccessPolicy.m */; };
+		693220361C16437100B07D98 /* AZSSharedAccessSignatureHelper.h in Headers */ = {isa = PBXBuildFile; fileRef = 693220301C16437100B07D98 /* AZSSharedAccessSignatureHelper.h */; };
+		693220371C16437100B07D98 /* AZSSharedAccessSignatureHelper.m in Sources */ = {isa = PBXBuildFile; fileRef = 693220311C16437100B07D98 /* AZSSharedAccessSignatureHelper.m */; };
+		6932203C1C16438600B07D98 /* AZSBlobContainerPermissions.h in Headers */ = {isa = PBXBuildFile; fileRef = 693220381C16438600B07D98 /* AZSBlobContainerPermissions.h */; };
+		6932203D1C16438600B07D98 /* AZSBlobContainerPermissions.m in Sources */ = {isa = PBXBuildFile; fileRef = 693220391C16438600B07D98 /* AZSBlobContainerPermissions.m */; };
+		6932203E1C16438600B07D98 /* AZSSharedAccessBlobParameters.h in Headers */ = {isa = PBXBuildFile; fileRef = 6932203A1C16438600B07D98 /* AZSSharedAccessBlobParameters.h */; };
+		6932203F1C16438600B07D98 /* AZSSharedAccessBlobParameters.m in Sources */ = {isa = PBXBuildFile; fileRef = 6932203B1C16438600B07D98 /* AZSSharedAccessBlobParameters.m */; };
+		693220431C16439500B07D98 /* AZSSasTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 693220401C16439500B07D98 /* AZSSasTests.m */; };
+		693220441C16439500B07D98 /* AZSTestSemaphore.m in Sources */ = {isa = PBXBuildFile; fileRef = 693220421C16439500B07D98 /* AZSTestSemaphore.m */; };
+		693220461C16439D00B07D98 /* Resources in Resources */ = {isa = PBXBuildFile; fileRef = 693220451C16439D00B07D98 /* Resources */; };
+		693EA07F1BCDB54000507221 /* (null) in Headers */ = {isa = PBXBuildFile; };
```

Comment:
I think we can remove these null PBXBuildFiles, as long as everything still looks good in XCode.  Feel free to do this in a future commit